samples/drivers/espi/src/main.c

```diff
@@ -12,12 +12,15 @@
 #include <drivers/gpio.h>
 #include <drivers/espi.h>
 
-#ifdef CONFIG_ESPI_GPIO_DEV_NEEDED
+#ifdef DT_GPIO_ESPI_PINS_PWRG_GPIOS_CONTROLLER
 static struct device *gpio_dev0;
```

Comment:
It would be nice if this were `pwrg_gpio_dev` (similarly `rsm_gpio` below) so maintainer's don't have to guess what the numbering indicates.